share/completions/clj.fish

```diff
@@ -0,0 +1,79 @@
+set bb_helper '
+    ;; ht Bob B@Clojurians Slack 
+    ;; https://raw.githubusercontent.com/bobisageek/bb-scripts/main/clj-alils.clj
+    ;; list all aliases available from the current context
+    ;; based on :config-files from `clojure -Sdescribe` (really deps.clj instead of clojure)
+
+(require \'[babashka.fs :as fs])
+
+(def config (->> (with-out-str (babashka.tasks/clojure "-Sdescribe"))
+                 clojure.edn/read-string))
+
+(defn configs []
+  (reverse (:config-files config)))
+
+(defn aliases-from-config [path]
+  (->> (clojure.edn/read-string (slurp path))
+       :aliases
+       keys
+       (map name)
+       sort))
+
+(defn installed-tools
+  []
+  (as-> config $
+    (:config-dir $)
+    (str $ "/tools/")
+    (fs/match $ "regex:.*\\\\.edn")
+    (map fs/file-name $)
+    (map #(fs/strip-ext % {:ext "edn"}) $)))
+
+(defn aliases []
+  (->> (configs)
+       (mapcat aliases-from-config)
+       dedupe))
+
+(run! println
+      (case (first *command-line-args*)
+        "tools" (installed-tools)
+        (aliases)))'
+
+function __fish_clj_aliases
+    which bb >/dev/null 2>&1; or return 1
+    bb (echo $bb_helper | psub)
```

Comment:
Does `bb` take input from stdin?

If so, this could just be

```fish
echo $bb_helper | bb
```

or possibly `echo $bb_helper | bb -`.

Reply:
It can, but it will make noise:

```
❯ echo "(+ 1 2)" | bb -i
Babashka v0.10.163 REPL.
Use :repl/quit or :repl/exit to quit the REPL.
Clojure rocks, Bash reaches.

user=> 3
user=> ⏎                                                                                                                                                                                                          
~
❮ bb "(+ 1 2)"
3

~
❯
```

So I guess `psub` is needed?